examples/config_dc2_steps.yaml

```diff
@@ -0,0 +1,121 @@
+# This is an example of the template specification file for Steps used in DC2 campaigns
+
+# Import the common element templates
+- import: "${CM_CONFIGS}/config_standard_elements.yaml"
+# Now define the steps
+- SpecBlock:
+      name: dc2_step1
+      includes: ['step']
+      data:
```

Comment:
What kind of category is `data` here? I see `pipeline_yaml` but what other information might we want in `data` ?

Reply:
Anything that gets passed down the heirarchy that isn't collection name templates (that goes in collections), configuration of how to build child nodes (that goes in child_config) or renaming what spec blocks to use (that goes in spec_aliases)